pkg/eks/update.go

```diff
@@ -91,6 +91,37 @@ func (c *ClusterProvider) UpdateClusterConfigForLogging(cfg *api.ClusterConfig)
 	return nil
 }
 
+// GetCurrentClusterConfigForEndpoints fetches current cluster endpoint configuration for public and private access types
+func (c *ClusterProvider) GetCurrentClusterConfigForEndpoints(spec *api.ClusterConfig) (*bool, *bool, error) {
```

Comment:
i don't think this should return pointers unless they can be `nil` but then one actually needs to verify they aren't `nil` before dereferencing (which currently isn't done)

Reply:
Done.